core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -180,13 +198,21 @@ func (c *coordinator) ReportBlocks(
 	maxBlocks, // TODO: unused for now
 	maxCallbacks int, // TODO: unused for now
 ) (blocks []ocr2vrftypes.Block, callbacks []ocr2vrftypes.AbstractCostedCallbackRequest, err error) {
+	// Instantiate the gas used by this batch.
+	currentBatchGasLimit := coordinatorOverhead
+
 	// TODO: use head broadcaster instead?
 	currentHeight, err := c.lp.LatestBlock(pg.WithParentCtx(ctx))
 	if err != nil {
 		err = errors.Wrap(err, "header by number")
 		return
 	}
 
+	// Evict expired items from the cache.
+	now := time.Duration(time.Now().Unix() * int64(time.Second))
```

Comment:
This doesn't look right. `now` can't be a duration, it has to be a `time.Time`. You can get current UTC time via:

```suggestion
	now := time.Now().UTC()
```